### PR TITLE
GitHub actions fixes

### PR DESCRIPTION
**Pull request by bjaglin**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - main
```
Comment (liancheng) [inline .github/workflows/ci.yaml:6]: Hmm, we don't have a `main` branch?

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: olafurpg/setup-scala@v11
       - uses: coursier/cache-action@v6
       - name: Test
-        run: sbt scalafmtCheck "rules/scalafix --check" +test
+        run: sbt scalafmtCheck "rules2_12/scalafix --check" +test
```
Comment (liancheng) [inline .github/workflows/release.yaml:17]: Thanks for the fix!

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (bjaglin) [reply .github/workflows/ci.yaml]: oops, bad copy/paste... amended!

**Review by liancheng — APPROVED** [review]
Thanks, merging!

**Pull request merged**